workflows/methylseq.nf

```diff
@@ -234,6 +250,34 @@ workflow METHYLSEQ {
         multiqc_report = MULTIQC.out.report.toList()
         ch_versions    = ch_versions.mix(MULTIQC.out.versions)
     }
+
+    //
+    // MODULE: METHLYKIT_REPORT
+    //
+
+    ch_methylkit_report = Channel.empty()
+    ch_samplesheet = Channel.value(file(params.input, checkIfExists: true))
+
+    if (!params.skip_methylkit) {
+
+        bismark_cov_dir = Channel.fromPath("${params.outdir}/bismark/methylation_calls/methylation_coverage/")
+        METHYLKIT(
+            QUALIMAP_BAMQC.out.results.collect(),
```

Comment:
Are we sure QUALIMAP_BAMQC always finishes after the bismark .cov files are generated?

If not, it might be better to explicitly wait until the .cov files are all generated.
This appears to be the module to collect() the .cov.gz files from:
https://github.com/epicrispr-biotechnologies/epic-methylseq/blob/ee74616ec422338cf47b0fd8b75f204d2d2e9ae4/modules/nf-core/bismark/methylationextractor/main.nf